src/main/java/org/folio/des/scheduling/quartz/converter/bursar/ExportConfigToBursarTriggerConverter.java

```diff
@@ -0,0 +1,108 @@
+
+  @Value("${folio.quartz.bursar.timeZone}")
+  private String timeZone;
+
+  @Override
+  public ExportTrigger convert(@NotNull ExportConfig exportConfig) {
+
+    ScheduleParameters scheduleParameters = createBursarScheduleParameters(exportConfig);
+
+    return new ExportTrigger(false, scheduleParametersToTriggerConverter
+      .convert(scheduleParameters, getTriggerGroup(exportConfig)));
+  }
+
+  private ScheduleParameters createBursarScheduleParameters(ExportConfig exportConfig) {
+
+    ExportConfig.SchedulePeriodEnum schedulePeriod = exportConfig.getSchedulePeriod();
+
+    if (ExportConfig.SchedulePeriodEnum.NONE.equals(schedulePeriod)) return null;
+
+    return switch (schedulePeriod) {
+      case HOUR -> buildHourlyScheduleParam(exportConfig);
+      case DAY -> buildDailyScheduleParam(exportConfig);
+      case WEEK -> buildWeeklyScheduleParam(exportConfig);
+      default -> null;
+    };
+  }
+
+  private ScheduleParameters buildWeeklyScheduleParam(ExportConfig exportConfig) {
+    log.info("Inside buildWeeklyScheduleParam");
+    ScheduleParameters scheduleParameters = new ScheduleParameters();
+    scheduleParameters.setScheduleFrequency(exportConfig.getScheduleFrequency());
+    scheduleParameters.setTimeZone(timeZone);
+    scheduleParameters.setSchedulePeriod(ScheduleParameters.SchedulePeriodEnum.valueOf(exportConfig.getSchedulePeriod().name()));
+    scheduleParameters.setScheduleTime(OffsetTime.parse(exportConfig.getScheduleTime()).toLocalTime().toString());
+    scheduleParameters.setWeekDays(createWeekDaysEnum(exportConfig.getWeekDays()));
+
+    return scheduleParameters;
+  }
+
+  private ScheduleParameters buildDailyScheduleParam(ExportConfig exportConfig) {
+    log.info("Inside buildDailyScheduleParam");
+    ScheduleParameters scheduleParameters = new ScheduleParameters();
+    scheduleParameters.setScheduleFrequency(exportConfig.getScheduleFrequency());
+    scheduleParameters.setTimeZone(timeZone);
+    scheduleParameters.setSchedulePeriod(ScheduleParameters.SchedulePeriodEnum.valueOf(exportConfig.getSchedulePeriod().name()));
+    scheduleParameters.setScheduleTime(OffsetTime.parse(exportConfig.getScheduleTime()).toLocalTime().toString());
+    return scheduleParameters;
+  }
+
+  private ScheduleParameters buildHourlyScheduleParam(ExportConfig exportConfig) {
+    log.info("Inside buildHourlyScheduleParam");
+    ScheduleParameters scheduleParameters = new ScheduleParameters();
+    scheduleParameters.setScheduleFrequency(exportConfig.getScheduleFrequency());
+    scheduleParameters.setTimeZone(timeZone);
+    scheduleParameters.setSchedulePeriod(ScheduleParameters.SchedulePeriodEnum.valueOf(exportConfig.getSchedulePeriod().name()));
+    setHourlyScheduledTime(scheduleParameters);
+
+    return scheduleParameters;
+  }
+
+  private void setHourlyScheduledTime(ScheduleParameters scheduleParameters) {
+    final ZonedDateTime now = ZonedDateTime.now(ZoneId.of(timeZone));
```

Comment:
https://github.com/folio-org/mod-data-export-spring/blob/master/src/main/java/org/folio/des/scheduling/ExportTrigger.java#L103 

Maybe it should be left it according to prior logic, were scheduling was started from next hour

Reply:
done